README.md

```diff
@@ -13,7 +13,7 @@
 
 This deployment follows Dex by CoreOS & Kubernetes Documentations:
 
-* [Kubernetes OIDC Doc](https://kubernetes.io/docs/admin/authentication/#option-1---oidc-authenticator)
+* [Kubernetes OIDC Doc](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#option-1-oidc-authenticator)
```

Comment:
I think we should change for https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens page ref (this is about OIDC workflow, not only kubectl configuration).

Reply:
Done